### DISCUSSION TITLE
Windows runners support on ARC v2

### DISCUSSION THREAD

**Discussion by nascit** [community/community#62477] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi,

Does ARC have on its backlog support for WIndows runners?

Context: We have been using legacy ARC for over a year and using Windows runners configured as per docs:
https://github.com/actions/actions-runner-controller/blob/master/docs/configuring-windows-runners.md

Haven't seen any mention on Windows runners on ARC v2 docs though.

**Comment by DrShushen** [comment]
This is not currently officially supported, but there are workarounds: https://github.com/actions/actions-runner-controller/discussions/2880#

I think lack of official Windows Runner support should be made clearer in the documentation.

Supporting Windows Runners in future would certainly be a huge plus, in my view.
